### PR TITLE
Carbamazepine PBPK model developed by SPM

### PR DESCRIPTION
**Pull request by AndreDlm**
Independently developed model by SPM

@Yuri05

### CODE REVIEW

File: Evaluation/Input/Content/Concentration_time_profiles.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Simulated versus observed concentration-time profiles of all data listed in [Section 2.2.2](#222-Clinical-Data) are presented below.
```
Comment (Yuri05) [inline Evaluation/Input/Content/Concentration_time_profiles.md:1]: change the link to low case
`[Section 2.2.2](#222-clinical-data)`

File: Evaluation/Input/Content/GOF_diagnostics.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Below you find the goodness-of-fit visual diagnostic plots for the PBPK model performance of all data used presented in [Section 2.2.2](#222-Clinical-Data).
```
Comment (Yuri05) [inline Evaluation/Input/Content/GOF_diagnostics.md:1]: same

**Comment by Yuri05** [comment]
Here source should be **Publication** and not **Unknown**??

<details><summary><b>Carbamazepin compound BB </b></summary>

grafik

grafik
</details>

<details><summary><b>[15N]- Carbamazepin compound BB </b></summary>

grafik

grafik

</details>

**Comment by Yuri05** [comment]
Missing value origin (both Compound -BBs)
grafik

File: Evaluation/Input/Content/References.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+**Wada 1978** Wada, J. A., Troupin, A. S., Friel, P., Remick, R., Leal, K., & Pearmain, J. (1978). Pharmacokinetic comparison of tablet and suspension dosage forms of carbamazepine. *Epilepsia*, *19*(3), 251-255.
+
+**Williamson 2016** Williamson, B., Lorbeer, M., Mitchell, M. D., Brayman, T. G., & Riley, R. J. (2016). Evaluation of a novel PXR‐knockout in HepaRG™ cells. *Pharmacology research & perspectives*, *4*(5), e00264.
+
+**Willmann 2007** Willmann, S., Höhn, K., Edginton, A., Sevestre, M., Solodenko, J., Weiss, W., ... & Schmitt, W. (2007). Development of a physiology-based whole-body population model for assessing the influence of individual variability on the pharmacokinetics of drugs. *Journal of pharmacokinetics and pharmacodynamics*, *34*(3), 401-431.
+
+**Wong 1983** Wong, Y. Y., Ludden, T. M., & Bell, R. D. (1983). Effect of erythromycin on carbamazepine kinetics. *Clinical Pharmacology & Therapeutics*, *33*(4), 460-464.
+
+
+
```
Comment (Yuri05) [inline Evaluation/Input/Content/References.md:146]: I cannot find the following references used in the model:
grafik

File: Carbamazepine-Model.json [hunk 4]
```diff
@@ -10681,8 +10708,7 @@
               "Value": 0.243,
               "ValueOrigin": {
                 "Source": "Publication",
-                "Method": "InVitro",
-                "Description": "Morselli 1975"
+                "Description": "Morselli 1975 (DOI: 10.1007/978-3-642-85921-2_16)"
```
Comment (Yuri05) [inline Carbamazepine-Model.json:10711]: Method was inVitro and now empty - on purpose?

File: Carbamazepine-Model.json [hunk 5]
```diff
@@ -10698,17 +10724,15 @@
               "Unit": "mg/l",
               "ValueOrigin": {
                 "Source": "Publication",
-                "Method": "InVivo",
-                "Description": "Mean value of the following FaHIF solubility data reported in the literature: 336 µg/mL, pH 6.2 (Annaert 2010); 283 µg/mL, pH 7.0 (Söderlind 2010); 306 mg/mL, pH 6.9 (Clarysse 2011)"
+                "Description": "Mean value of the following FaHIF solubility data reported in the literature: 336 µg/mL, pH 6.2 (Annaert 2010; DOI: 10.1016/j.ejps.2009.10.005); 283 µg/mL, pH 7.0 (Söderlind 2010; DOI: 10.1021/mp100144v); 306 mg/mL, pH 6.9 (Clarysse 2011; DOI: 10.1016/j.ejps.2011.04.016)"
```
Comment (Yuri05) [inline Carbamazepine-Model.json:10727]: Method was inVitro and now empty - on purpose?

File: Carbamazepine-Model.json [hunk 6]
```diff
@@ -10698,17 +10724,15 @@
...
               }
             },
             {
               "Name": "Reference pH",
               "Value": 6.7,
               "ValueOrigin": {
                 "Source": "Publication",
-                "Method": "InVivo",
-                "Description": "Mean value of the following FaHIF solubility data reported in the literature: 336 µg/mL, pH 6.2 (Annaert 2010); 283 µg/mL, pH 7.0 (Söderlind 2010); 306 mg/mL, pH 6.9 (Clarysse 2011)"
+                "Description": "Mean value of the following FaHIF solubility data reported in the literature: 336 µg/mL, pH 6.2 (Annaert 2010; DOI: 10.1016/j.ejps.2009.10.005); 283 µg/mL, pH 7.0 (Söderlind 2010; DOI: 10.1021/mp100144v); 306 mg/mL, pH 6.9 (Clarysse 2011; DOI: 10.1016/j.ejps.2011.04.016)"
```
Comment (Yuri05) [inline Carbamazepine-Model.json:10735]: same here

File: Carbamazepine-Model.json [hunk 7]
```diff
@@ -10984,34 +11018,56 @@
               "Value": 0.243,
               "ValueOrigin": {
                 "Source": "Publication",
-                "Method": "InVitro",
-                "Description": "Morselli 1975"
+                "Description": "Morselli 1975 (DOI: 10.1007/978-3-642-85921-2_16)"
```
Comment (Yuri05) [inline Carbamazepine-Model.json:11021]: same here

File: Carbamazepine-Model.json (hunk 4 above)
Comment (AndreDlm) [reply Carbamazepine-Model.json:10711]: Difficult to classify in my opinion; blood was sampled *in vivo* and further processed and analyzed *in vitro*, not sure how to best call that...

**Pull request merged**